plugins/colorschemes/kanagawa-paper.nix

```
@@ -0,0 +1,230 @@
+{
+  lib,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+  inherit (lib) types;
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "kanagawa-paper";
+  isColorscheme = true;
+  packPathName = "kanagawa-paper.nvim";
+  package = "kanagawa-paper-nvim";
+
+  description = ''
+    You can select the theme in two ways:
+    - Set `colorschemes.kanagawa-paper.settings.theme` AND explicitly unset `vim.o.background` (i.e. `options.background = ""`).
```

I believe nixvim's `vim.o` option is `opts` not `options`

```suggestion
    - Set `colorschemes.kanagawa-paper.settings.theme` AND explicitly unset `vim.o.background` (i.e. `opts.background = ""`).
```

---

A tip regarding options:

If you can get a reference to the _actual_ option struct, you can stringify it to it's human-readable opition path.

E.g. `"${options.colorschemes.kanagawa-paper.settings}"` will stringify to `colorschemes.kanagawa-paper.settings` or `programs.nixvim.colorschemes.kanagawa-paper.settings`, depending on context.

Note this only works for _actual_ options, not sub-attrs within a freeform option or sets that contain options (e.g. `options.colorschemes.kanagawa-paper` is not an option, neither is `options.colorschemes.kanagawa-paper.settings.foo.bar`).

For sub-attrs you can uaually just type that bit manually though: e.g. `"${options.colorschemes.kanagawa-paper.settings}.foo.bar"`

The advantage of this approach are a) you get an error if you have a typo b) you get the correct context-dependant prefix in warnigns (doesn't usually affect docs).